rcgen/src/lib.rs

```diff
@@ -73,6 +73,7 @@ pub type RcgenError = Error;
 pub struct Certificate {
 	params: CertificateParams,
 	key_pair: KeyPair,
+	spki: Vec<u8>,
```

Comment:
Nit: I feel like SPKI is not a very accessible acronym, how about writing it out as `subject_public_key_info` in the field name and the type constructors at least (I probably wouldn't expand it in `pub_key_spki`)?

Reply:
Good call. Fixed.